ccp/modules/exporter-compose.yml

```diff
@@ -65,3 +65,8 @@ services:
       - "traefik.http.routers.reporter_ccp.tls=true"
       - "traefik.http.middlewares.reporter_ccp_strip.stripprefix.prefixes=/ccp-reporter"
       - "traefik.http.routers.reporter_ccp.middlewares=reporter_ccp_strip"
+
+  focus:
+    environment:
+      EXPORTER_URL: "http://exporter:8092"
+      AUTH_HEADER: "${EXPORTER_API_KEY}"
```

Comment:
Is this set correctly? It says "header" in the key but the value only has an API key. [focus docs](https://github.com/samply/focus) state that there should be "ApiKey " in front

Reply:
@djuarezgf and I have agreed on the ApiKey type of the authentication some year ago. Is there a reason to have a different type?

Reply:
I looked into Focus' code: The Exporter uses a different header field (x-api-key) , so it is generally handled differently. This could be something to align in A) Explorter b) our other components using API-Keys. At the very least, this could and should be mentioned in Focus docs to be clear.

Reply:
@enola-dkfz thanks for jogging my memory. David chose the header recommended by OpenAPI, we chose the header recommended by the RFC and used throughout our components. Could you or @davidmscholz add a brief sentence to the focus documentation, that if the endpoint is exporter the semantics of the auth_header option changes?

Reply:
Oh that's complicated, because when the target app is Exporter, the endpoint type is actually Blaze (or Blaze and SQL). As you said, Focus has outgrown its architecture.

Reply:
Thanks for adressing that in https://github.com/samply/focus/pull/185